open_academy/wizards/wizard_save_attendee.py

```diff
@@ -0,0 +1,15 @@
+from odoo import models, fields, Command
+
+
+class Wizard(models.TransientModel):
+    _name = 'wizard_save_attendee'
```

Comment:
change all the references of the name of the wizard from `wizard_save_attendee` to `assign.attendee.sessions`
modes `_name` don't use an underscore in the name, uses `.` , change this when you rename the wizard